### PR TITLE
Community calendar added to community page

### PR DESCRIPTION
**Pull request by nathenf**
- Added a view of the calendar to community page
- Included a link to subscribe to the community calendar

#856

### CODE REVIEW

**Comment by nathenf** [comment]
Hey @olayway I've tried to integrate google calendar into this page and it builds but when I check the preview I get: 

_Application error: a client-side exception has occurred (see the browser console for more information)._ 

I can't figure out why or what this means. Could you assist? 🙏

File: content/community.md [hunk 1]
```diff
@@ -7,7 +7,15 @@ created: 2023-08-30
...
 
-[Join our community chat here](https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA)
+👉 [Join our community chat here](https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA) 
+
+## Upcoming community events
+
+Check out the calendar below to join specific events, or if you want to have all the Life Itself community events in your calendar you can subscribe to the entire calendar! 🎉
+
+👉 [Subscribe to the community calendar](https://calendar.google.com/calendar/u/0?cid=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
+<iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=2&bgcolor=%23f0ca5e&ctz=Africa%2FCeuta&hl=en_GB&showPrint=0&showTitle=0&showCalendars=0&src=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23E4C441" style="border-width:0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```
Comment (rufuspollock) [inline content/community.md:17]: I suspect this is the source of the problem in that you have to use React style stuff when doing html  ...

Specifically style is tricky and i think you need to do it as a property i.e. like this

```
style={{border-width: "0"}}
```

```
<iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=2&bgcolor=%23f0ca5e&ctz=Africa%2FCeuta&hl=en_GB&showPrint=0&showTitle=0&showCalendars=0&src=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23E4C441" style={{border-width: "0"}} width="800" height="600" frameborder="0" scrolling="no"
```

For background see https://github.com/datopian/flowershow/issues/508#issuecomment-1559970929

File: content/community.md [hunk 2]
```diff
@@ -7,7 +7,15 @@ created: 2023-08-30
...
 _A community for people, who recognise the meta-crises we face, know there is a possibility of a new paradigm and are ready to do something about it!_
 
-[Join our community chat here](https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA)
+👉 [Join our community chat here](https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA) 
+
+## Upcoming community events
+
+Check out the calendar below to join specific events, or if you want to have all the Life Itself community events in your calendar you can subscribe to the entire calendar! 🎉
+
+👉 [Subscribe to the community calendar](https://calendar.google.com/calendar/u/0?cid=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
```
Comment (rufuspollock) [inline content/community.md:16]: I'd recommend a blank line after this before the html ...

**Review by rufuspollock — CHANGES_REQUESTED** [review]
I think i've found a fix for you!

**Comment by rufuspollock** [comment]
FIXED ... finally. https://embed-calendar-community-cal.lifeitself-org.pages.dev/community

image

**Pull request merged**